lib/stdlib/test/digraph_utils_SUITE.erl

```diff
@@ -153,6 +153,8 @@ subgraph(Config) when is_list(Config) ->
     {fg,f,g,fgl} = digraph:edge(SG, fg),
     {fg2,f,g,fgl2} = digraph:edge(SG, fg2),
     {_, {_, acyclic}} = lists:keysearch(cyclicity, 1, digraph:info(SG)),
+    digraph:add_edge(SG, f, g),
```

Comment:
This doesn't fail if I remove the bug fix. You will need to test the return value.